VMEncryption/main/test/test_resource_disk_util.py

```diff
@@ -39,7 +39,7 @@ def setUp(self):
         self.mock_crypt_mount_config_util = mock.create_autospec(CryptMountConfigUtil)
         self.mock_passphrase_filename = "mock_passphrase_filename"
         mock_public_settings = {}
-        self.resource_disk = ResourceDiskUtil(self.logger, self.mock_disk_util, self.mock_crypt_mount_config_util, self.mock_passphrase_filename, mock_public_settings, ["ubuntu", "16"])
+        self.resource_disk = ResourceDiskUtil(self.logger, self.mock_disk_util, self.mock_crypt_mount_config_util, self.mock_passphrase_filename, mock_public_settings, ["ubuntu", "16"],False)
```

Comment:
Will it be possible to add unit test for CVM scenarios to? Not a blocker for this PR but if possible then add in next PR.